packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -30,6 +30,59 @@ import 'tooltip_theme.dart';
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=EeEfD5fI-5Q}
 ///
+/// ## Example
+///
```

Comment:
Probably no need for an "Example" subheading, since the dartdoc itself will already create a code block and we don't usually have one for our code samples
```suggestion
```